terraform/environments/production/dns.tf

```diff
@@ -37,3 +37,10 @@ module "dutyfree_address" {
   dns_zone  = var.dns_zone
   subdomain = "resource-types"
 }
+
+module "mailgun_records" {
+  source = "../../dependencies/mailgun"
+
+  dns_zone     = var.dns_zone
+  verification = "k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCiXNfw2u/anSOEvASvp94Rt+sp4ujO/AETVVtnqT3MC3By1EZBD8b0JowMZZKGkT9RwstC55fM66hhuJ0Az0lUZql2hex96k3mZjNUghaMoiPbIxa7bjYI2+vVEGIq5Y95PUJnKZMqEhe4WTob+vWGBsmKxHTr0lueShiO9fXFfwIDAQAB"
```

Comment:
Just confirming this is meant to be public

Reply:
Yep - it's being set in a public DNS record for Mailgun to verify control of the domain, so it'll be public whether it's committed or not. We had the same question. :P